llama_toolchain/cli/stack/configure.py

```diff
@@ -31,49 +35,93 @@ def __init__(self, subparsers: argparse._SubParsersAction):
         self.parser.set_defaults(func=self._run_stack_configure_cmd)
 
     def _add_arguments(self):
-        from llama_toolchain.core.distribution_registry import (
-            available_distribution_specs,
-        )
-        from llama_toolchain.core.package import ImageType
-
-        allowed_ids = [d.distribution_type for d in available_distribution_specs()]
         self.parser.add_argument(
             "config",
             type=str,
-            help="Path to the package config file (e.g. ~/.llama/builds/<distribution>/<image_type>/<name>.yaml)",
+            help="Path to the build config file (e.g. ~/.llama/builds/<image_type>/<name>-build.yaml). For docker, this could also be the name of the docker image. ",
         )
 
     def _run_stack_configure_cmd(self, args: argparse.Namespace) -> None:
         from llama_toolchain.core.package import ImageType
 
-        config_file = Path(args.config)
-        if not config_file.exists():
-            self.parser.error(
-                f"Could not find {config_file}. Please run `llama stack build` first"
+        build_config_file = Path(args.config)
+        if not build_config_file.exists():
+            cprint(
+                f"Could not find {build_config_file}. Trying docker image name instead...",
+                color="green",
             )
-            return
-
-        configure_llama_distribution(config_file)
 
+            build_dir = (
+                Path(os.path.expanduser("./.llama/distributions"))
+                / ImageType.docker.value
+            )
+            build_config_file = build_dir / f"{args.config}-build.yaml"
 
-def configure_llama_distribution(config_file: Path) -> None:
-    from llama_toolchain.common.serialize import EnumEncoder
-    from llama_toolchain.core.configure import configure_api_providers
+            os.makedirs(build_dir, exist_ok=True)
 
-    with open(config_file, "r") as f:
-        config = PackageConfig(**yaml.safe_load(f))
+            script = pkg_resources.resource_filename(
```

Comment:
I wonder if it is simpler to run configure _within_ the container? because otherwise we have created another copy of the build file now which can diverge from the one inside the container. 

so run `docker run configure.sh` and that will read the build, and dump the created config on stdout and then that can be saved outside the container.